deploy/olm-catalog/kubevirt-hyperconverged/1.4.0/hco00.crd.yaml

```diff
@@ -151,9 +151,11 @@ spec:
                 description: BareMetalPlatform indicates whether the infrastructure is baremetal.
                 type: boolean
               featureGates:
-                additionalProperties:
-                  type: boolean
-                description: featureGates HyperConvergedFeatureGates contain a list of feature enabler flags. Setting a flag to `true` will enable the feature. Setting `false` or removing the feature gate, disables the feature. optional+
+                description: featureGates HyperConvergedFeatureGates contains a list of feature enabler flags. Setting a flag to `true` will enable the feature. Setting `false` or removing the feature gate, disables the feature.
+                properties:
+                  HotplugVolumes:
+                    description: Allow attaching a data volume to a running VMI; type is boolean; default is false
```

Comment:
Do we really need to put the type in the English text? There's the `type` section for that. I would also not document the default. A missing flags can means that the admin does not care, and trusts HCO to do the right thing; we may want to flip the default without much notice.